### PR TITLE
DSS Deployment: remove hardcoded parameter values

### LINKED ISSUE #29: Fix hard-coded SSM store names
In `chalice/build_deploy_config.sh` and `daemons/build_deploy_config.sh` we have a hard-coded SSM parameter store name "dcp" that should be using an environment variable instead.

Use `DSS_PARAMETER_STORE`

### PR DESCRIPTION
**Pull request by amarjandu**
get rid of hardcoded `dcp/dss`, use `DSS_PARAMETER_STORE`

closes #29

### CODE REVIEW

File: daemons/build_deploy_config.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ cat "$config_json" | jq ".stages.$stage.tags.DSS_DEPLOY_ORIGIN=\"$DEPLOY_ORIGIN\
 	.stages.$stage.tags.owner=\"${DSS_INFRA_TAG_OWNER}\" | \
 	.stages.$stage.tags.env=\"${DSS_DEPLOYMENT_STAGE}\""  | sponge "$config_json"
 
-env_json=$(aws ssm get-parameter --name /dcp/dss/${DSS_DEPLOYMENT_STAGE}/environment | jq -r .Parameter.Value)
+env_json=$(aws ssm get-parameter --name /${DSS_PLATFORM}/${DSS_DEPLOYMENT_STAGE}/environment | jq -r .Parameter.Value)
```
Comment (chmreid) [inline daemons/build_deploy_config.sh]:
```suggestion
env_json=$(aws ssm get-parameter --name /${DSS_PARAMETER_STORE}/${DSS_DEPLOYMENT_STAGE}/environment | jq -r .Parameter.Value)
```

File: chalice/build_deploy_config.sh [hunk 2]
```diff
@@ -54,7 +54,7 @@ cat "$config_json" | jq ".stages.$stage.tags.DSS_DEPLOY_ORIGIN=\"$DEPLOY_ORIGIN\
 	.stages.$stage.tags.owner=\"${DSS_INFRA_TAG_OWNER}\" | \
 	.stages.$stage.tags.env=\"${DSS_DEPLOYMENT_STAGE}\""  | sponge "$config_json"
 
-env_json=$(aws ssm get-parameter --name /dcp/dss/${DSS_DEPLOYMENT_STAGE}/environment | jq -r .Parameter.Value)
+env_json=$(aws ssm get-parameter --name /${DSS_PLATFORM}/${DSS_DEPLOYMENT_STAGE}/environment | jq -r .Parameter.Value)
```
Comment (chmreid) [inline chalice/build_deploy_config.sh]:
```suggestion
env_json=$(aws ssm get-parameter --name /${DSS_PARAMETER_STORE}/${DSS_DEPLOYMENT_STAGE}/environment | jq -r .Parameter.Value)
```

File: scripts/status.py [hunk 3]
```diff
@@ -19,9 +20,11 @@
 args = parser.parse_args()
 
 sm = boto3.client("secretsmanager")
+parameter_store = os.environ.get("DSS_PLATFORM")
```
Comment (chmreid) [inline scripts/status.py]:
```suggestion
parameter_store = os.environ.get("DSS_PARAMETER_STORE")
```

**Review by chmreid — CHANGES_REQUESTED** [review]
parameter store names will not be set correctly - see suggested changes for fixes

File: daemons/build_deploy_config.sh (hunk 1 above)
Comment (amarjandu) [reply daemons/build_deploy_config.sh]: >.> got carried away, thanks for catching that :)

**Review by chmreid — APPROVED** [review]

**Pull request merged**